AIM/AIM.xcodeproj/xcuserdata/oskarszydlowski.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,7 @@
 		<key>AIM.xcscheme</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>0</integer>
+			<integer>2</integer>
```

Comment:
This whole file should be in .gitignore

Reply:
I'll update gitignore later, have to finish sth